### DISCUSSION TITLE
Filecoin Artwork

### DISCUSSION THREAD

**Discussion by AOICARD** [filecoin-project/community#429] [General]
Hi all 🌹😊
We are honored to share with you our conceptual artwork of $FIL
The aspect ratio is 16:9 and you can use it as wallpaper for vertical screens like smartphones.
Using it is not prohibited by law and we would be happy if our artwork is used by people who appreciate it.❤
![File Coin](https://user-images.githubusercontent.com/96792705/147597879-52313187-f420-417a-a88f-da90c55bd850.jpg)

**Comment by jennijuju** [comment]
cool work! Thanks for sharing!

**Comment by kaitlin-beegle** [comment]
This is super cool!!!! 

I'm curious to know how this art was created....was it drawn?? It almost looks like a generative AI project!

**Reply by AOICARD** [reply]
Thank you!
It's full CGI artwork with the idea of using fingerprints as it is a symbol for individual identities and uniqueness.
